styles.css

```diff
@@ -49,6 +50,7 @@ body {
 .overlay {
   object-fit: cover;
   height: 100%;
+  position: absolute;
```

Comment:
It looks like your overlay <img> tag is sitting to the right of your cover-image and not on top of it. Adding `position: absolute;` will fix this.